hw1/02-palindrome.js

```diff
@@ -1,37 +1,37 @@
-const elem = document.querySelector("input");
-const result = document.querySelector("#result");
```

Comment:
The use of "document.getElementByID" is more efficient and is faster than "document.querySelector", if we are specifically targetting an element by its Id. So I would recommend using it instead of querySelector.